package.json

```diff
@@ -14,9 +14,10 @@
     "test:unit": "yarn cmsds test ./src",
```

Comment:
For a future story, might it make sense to include running tests locally before pushing our work up as well?

Reply:
I like that idea, yeah - can you add something to the Jira backlog to capture that @zarahzachz ?